picard/ui/mainwindow.py

```diff
@@ -1480,6 +1490,32 @@ def check_for_update(self, show_always):
             callback=update_last_check_date
         )
 
+    def make_script_selector_menu(self):
+        config = get_config()
+        naming_scripts = config.setting["file_renaming_scripts"]
+        selected_script_id = config.setting["selected_file_naming_script_id"]
+        self.script_quick_selector_menu.clear()
+
+        def _add_menu_item(title, id):
+            script_action = QtWidgets.QAction(title, self.script_quick_selector_menu)
+            script_action.triggered.connect(partial(self.select_new_naming_script, id))
+            if id == selected_script_id:
+                script_action.setIcon(self.style().standardIcon(QtWidgets.QStyle.SP_DialogApplyButton))
+            self.script_quick_selector_menu.addAction(script_action)
+
+        for (id, naming_script) in sorted(naming_scripts.items(), key=lambda item: item[1]['title']):
+            _add_menu_item(user_script_title(naming_script['title']), id)
+
+        # Add preset scripts not provided in the user-defined scripts list.
+        for script_item in get_file_naming_script_presets():
+            _add_menu_item(user_script_title(script_item['title']), script_item['id'])
```

Comment:
The presets show up for me as "User: Preset: The title". But in the combobox in options it works as expected and they only display as "Preset: The title".

Reply:
Definitely a copy'n'paste error.  Oops.  :blush: